libs/blocks/carousel/carousel.css

```diff
@@ -373,33 +373,48 @@ html[dir="rtl"] .carousel-slides .section.carousel-slide {
   width: unset;
 }
 
-.carousel.rounded-corners .carousel-slide,
-.carousel.rounded-corners .carousel-slide > div :is(p):has(> picture) img {
-  border-radius: var(--m-rounded-corners);
-}
-
-.carousel.l-rounded-corners .carousel-slide,
-.carousel.l-rounded-corners .carousel-slide > div :is(p):has(> picture) img {
-  border-radius: var(--l-rounded-corners);
+.carousel.video-fill .carousel-slide > div p > video,
+.carousel .carousel-slide.video-fill > div p > video {
+  width: 100%;
+  height: auto;
 }
 
 .carousel.s-rounded-corners .carousel-slide,
-.carousel.s-rounded-corners .carousel-slide > div :is(p):has(> picture) img {
+.carousel.s-rounded-corners .carousel-slide > div :is(p):has(> picture) img,
+.carousel.s-rounded-corners .carousel-slide > div :is(p):has(> video) video,
+.carousel.s-rounded-corners .carousel-slide > div :is(p):has(.milo-video) .milo-video {
```

Comment:
All these selectors look a bit over-engineered to me. Let's look at this part - `div :is(p):has(.milo-video) .milo-video`. Human-language this would be: look at a div that has a child that is a paragraph and has another `.milo-video` child inside and apply these properties to the `.milo-video` child. Which translated back into code would be `div p .milo-video`. Tried it on one of your test pages and it achieves the same result:
<img width="1512" alt="Screenshot 2024-11-27 at 14 28 15" src="https://github.com/user-attachments/assets/bacfe1e8-b9f2-4b00-bc7c-044c0155f35f">

So we could have this and the others as something like:
```suggestion
.carousel.s-rounded-corners .carousel-slide > div p > picture img,
.carousel.s-rounded-corners .carousel-slide > div p > video,
.carousel.s-rounded-corners .carousel-slide > div p .milo-video {
```

You could probably trim it down even further if you wish, something like:
```suggestion
.carousel.s-rounded-corners .carousel-slide > div :is(p > picture img, p > video, p .milo-video) {
```
but I didn't test the latter, you could give it a try.

Reply:
I wondered about that too. Thanks for the sanity check. I'll test and update accordingly.